app-backend/api/src/main/scala/annotation-project/AnnotationProjectTaskRoutes.scala

```diff
@@ -355,4 +385,33 @@ trait AnnotationProjectTaskRoutes
         }
       }
   }
+
+  def deleteTaskLabels(projectId: UUID, task: UUID): Route =
+    authenticate { user =>
+      authorizeScope(
+        ScopedAction(Domain.AnnotationProjects, Action.DeleteAnnotation, None),
+        user
+      ) {
+        authorizeAuthResultAsync {
+          AnnotationProjectDao
+            .authorized(
+              user,
+              ObjectType.AnnotationProject,
+              projectId,
+              ActionType.Annotate
+            )
+            .transact(xa)
+            .unsafeToFuture
+        } {
+          onSuccess(
+            AnnotationLabelDao
+              .deleteByProjectIdAndTaskId(projectId, task)
+              .transact(xa)
+              .unsafeToFuture
+          ) {
+            completeSomeOrNotFound
```

Comment:
I think this response could be clearer -- I'm not sure if we do this elsewhere, but a 404 for no labels in a task was surprising to me and made it seem like something was wrong with my request. It doesn't help that the message `akka-http` attached to the response was

> The requested resource could not be found but may be available again in the future.

Can we return a 200 with the count or a 204 instead? A particularly weird artifact of this response is that sending the same delete request twice in a row for a task can succeed the first time and return a 404 the second time.

Reply:
I think the same feedback should be applied to the other uses of `completeSomeOrNotFound` -- in `ProjectAnnotationRoutes` and `ProjectLayerAnnotationRoutes`